src/base/utils/misc.cpp

```diff
@@ -356,14 +356,22 @@ QString Utils::Misc::friendlyUnit(qint64 bytesValue, bool isSpeed)
         return QCoreApplication::translate("misc", "Unknown", "Unknown (size)");
     QString ret;
     if (unit == SizeUnit::Byte)
-        ret = QString::number(bytesValue) + " " + unitString(unit);
+        ret = QString::number(bytesValue) + QString::fromUtf8(C_NON_BREAKING_SPACE) + unitString(unit);
     else
-        ret = Utils::String::fromDouble(friendlyVal, 1) + " " + unitString(unit);
+        ret = Utils::String::fromDouble(friendlyVal, friendlyUnitPrecision(unit)) + QString::fromUtf8(C_NON_BREAKING_SPACE) + unitString(unit);
     if (isSpeed)
         ret += QCoreApplication::translate("misc", "/s", "per second");
     return ret;
 }
 
+int Utils::Misc::friendlyUnitPrecision(const SizeUnit &unit)
+{
+    // friendlyUnit's number of digits after the decimal point
```

Comment:
As to me, the comment would make more sense near the function declaration, especially taking into account that the function interprets "precision" term incorrectly (as well as `QString::number()`).